spec/unit/govuk_index/indexable_content_sanitiser_spec.rb

```diff
@@ -80,6 +82,32 @@
 
     expected_content = "title 1\n\nhello\n\ntitle 2\n\ngoodbye"
 
-    expect(expected_content).to eq(subject.clean(payload))
+    expect(subject.clean(payload)).to eq(expected_content)
+  end
+
+  context "Special character encoding" do
```

Comment:
"Special character encoding" makes it sound like this it could be a character encoding thing, but it's specifically HTML encoding we're opting out of.